secure-api-gateway-ig-extensions/pom.xml

```diff
@@ -75,8 +75,8 @@
         </dependency>
 
         <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-simple</artifactId>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
```

Comment:
Required to test MDC contains the x-fapi-interaction-id. The slf4j-simple logger uses NOPMDCAdapter which does nothing when we attempt to add items to the MDC.